src/woo_publications/publications/tests/test_document_api_endpoints.py

```diff
@@ -161,7 +157,10 @@ def test_list_documents(self):
                 "bestandsformaat": "unknown",
                 "bestandsnaam": "unknown.bin",
                 "bestandsomvang": 0,
-                "eigenaar": {"weergaveNaam": "groen", "identifier": "456"},
+                "eigenaar": {
+                    "identifier": self.organisation_member.identifier,
+                    "weergaveNaam": self.organisation_member.naam,
+                },
```

Comment:
can you please use the literal values from the constants for the audit headers? This indirection makes it much harder to read the test - looking purely at the code I need to context switch twice to figure out what the expected value is supposed to be.